Bug: http://github.com/hollie/misterhouse/issues/132
Author: Michael Stovenour <michael@stovenour.net>
Last-Update: 20130329
Description: Remove tracking scripts from user docs
 docs/toc.html contained two tracking scripts which no one on the project
 MisterHouse recalls. Scripts have been removed and can be added back in
 later if someone comes up with a patch that supports opt-in. The way the
 docs directory works every local MH copy was running the scripts as was
 the toc.html on sourceforge.net.

--- a/docs/toc.html
+++ b/docs/toc.html
@@ -25,15 +25,5 @@
 <br><a href="usage.html">Usage Map</a>
 <br><a href="authors.html">Authors</a>
 
-
-<script language="javascript"> var u=6416; var vsbl=0; var l=''; </script> <script type="text/javascript" language="Javascript" src="http://hitslog.com/counter.js"></script> <script language="javascript"> document.write('<a href=http://hitslog.com/><img border=0 width=1 height=1 src=http://hitslog.com/hl.php?'+l+'></a>'); </script> <noscript> <a href="http://hitslog.com/"><img border=0 width=1 height=1 src="http://hitslog.com/hl.php?u=6416" alt="create web counter"></a> </noscript>
-
-<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
-</script>
-<script type="text/javascript">
-_uacct = "UA-586835-4";
-urchinTracker();
-</script>
-
 </body>
 </html>
